.github/scripts/run-codespell.py

```diff
@@ -0,0 +1,33 @@
+"""
+ Copyright (C) 2023-2024 Intel Corporation
+
+ Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""
+
+import subprocess  # nosec B404
+import os
+
+
+def codespell_scan():
+    try:
+        codespell_result = subprocess.run(  # nosec
+            [
+                "codespell",
+                "-H",
+                "--quiet-level=3",
+                "--skip=*.supp,./.venv,./.git,./.github/scripts/run-codespell.py,./.github/workflows/.spellcheck-conf.toml",
```

Comment:
`"--skip=./.git,./.venv,./.github/workflows/.spellcheck-conf.toml,./.github/scripts/run-codespell.py",` is enough

Reply:
Done